assets/scss/_navbar_project.scss

```
@@ -260,4 +260,145 @@ $ecosystem-box-glow: inset 0 0em 4em #ebc01766, 0 0 0 2px #ebc01766, 0.3em 0.3em
+    outline: 2px solid #00b39f;
+    outline-offset: 3px;
+  }
+}
+
+// Light theme state for toggle button
+[data-theme="light"] .theme-toggle,
+[data-bs-theme="light"] .theme-toggle {
+  --ray-size: calc(var(--size) * -0.4);
+  --offset-orthogonal: calc(var(--size) * 0.65);
+  --offset-diagonal: calc(var(--size) * 0.45);
+  transform: scale(0.8);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)